runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterTable/UsersRegisterTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { VFC } from 'react';
+
+import { Grid, Card } from '@mui/material';
+import { DataGrid, GridInputSelectionModel } from '@mui/x-data-grid';
+import { useSelector } from 'react-redux';
+
+import { usersSelector } from '#src-app/store/slices/Users';
+
+import { ROWS_PER_PAGE } from '../../UsersBrowseView/UsersBrowseView.utils';
+import { DataGridStyle} from './UsersRegisterTable.styles';
+import useUsersRegisterColumns from './useUsersRegisterColumns';
+
+interface UsersRegisterTableProps {
+    page: number;
+    onPageChange: (page: number) => void;
+    pageSize: number;
+    onPageSizeChange: (pageSize: number) => void;
+    selections: GridInputSelectionModel;
+    handleSelectionChange: (selection: object) => void;
+    handleSelectedRolesChange: (id: number, value: string) => void;
+}
+
+const UsersRegisterTable: VFC<UsersRegisterTableProps> = ({
+    page,
+    onPageChange,
+    pageSize,
+    onPageSizeChange,
+    selections,
+    handleSelectionChange,
+    handleSelectedRolesChange
+}) => {
+    const userRegisterColumns = useUsersRegisterColumns(handleSelectedRolesChange);
+    const { notActivated, loading } = useSelector(usersSelector);
+
+    return (
+        <Card>
+            <Grid container>
+                <Grid item xs={12} md={12}>
+                    <DataGrid
+                        sx={DataGridStyle}
+                        autoHeight
+                        columns={userRegisterColumns}
+                        rows={notActivated.allByPage?.content ?? []}
+                        rowCount={notActivated.allByPage?.totalElements ?? 0}
+                        loading={loading}
+                        page={page}
+                        onPageChange={(newPage) => onPageChange(newPage)}
+                        pageSize={pageSize}
+                        onPageSizeChange={(newPageSize) => onPageSizeChange(newPageSize)}
+                        checkboxSelection={true}
+                        onSelectionModelChange={(e) => handleSelectionChange(e)}
+                        selectionModel={selections}
+                        paginationMode='server'
+                        rowsPerPageOptions={ROWS_PER_PAGE}
+                    />
```

Comment:
what is displayed if there are no pending user acceptance requests? is table visible? there should be message like in process run tab
<img width="807" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/d5066fdf-7727-4962-8079-bffddec3e3cf">

e.g. No pending user requests/no users waiting for acceptance

Reply:
It is displayed: No rows.